napari/layers/volume/tests/test_volume.py

```diff
@@ -94,6 +94,21 @@ def test_name():
     assert layer.name == 'img'
 
 
+def test_spacing():
+    """Test instantiating anisotropic 3D volume."""
+    shape = (10, 15, 20)
+    spacing = [3, 1, 1]
+    full_shape = tuple(np.multiply(shape, spacing))
```

Comment:
I'm really confused about this... Does this mean that the data is getting multiplied?

Reply:
yeah - this is maybe not so good, the data is not getting multiplied, but i needed to increase the "shape" so that the camera setting now gets the right centroid and field of view. It's not ideal, I need to think about how this can maybe get better ...

Reply:
Ooohhh... Hm. Ok I'll think about it also...

Reply:
First thought: this is the difference between our data coordinates and our "real world" coordinates, so we now need to start thinking about those, and maybe use `shape` and `real_shape`, or somesuch?

Reply:
yup agreed - i think that's the long term solution. for now this will be ok on just the volume layer